msg/tools/generate_microRTPS_bridge.py

```diff
@@ -94,6 +96,8 @@ def get_absolute_path(arg_parse_dir):
                     help="uORB templates dir, by default msg_dir/templates/uorb_microcdr", default=default_uorb_templates_dir)
 parser.add_argument("-q", "--urtps-templates-dir", dest='urtps_templates', type=str,
                     help="uRTPS templates dir, by default msg_dir/templates/urtps", default=default_urtps_templates_dir)
+parser.add_argument("-y", "--rtps-ids-file", dest='yaml_file', type=str,
+                    help="RTPS msg IDs definition file, by default tools/uorb_rtps_message_ids.yaml", default=default_rtps_id_file)
```

Comment:
It should be made clear that this is relative to the `msg` folder

Reply:
done